web/viewer.html

```diff
@@ -509,11 +509,11 @@
             <div id="markAsDecorative">
               <div class="radio">
                 <div class="radioButton">
-                  <input type="radio" id="decorativeButton" name="altTextOption">
+                  <input type="radio" id="decorativeButton" name="altTextOption" aria-describedby="decorativeDescription">
                   <label for="decorativeButton" data-l10n-id="editor_alt_text_mark_decorative_label">Mark as decorative</label>
                 </div>
                 <div class="radioLabel">
-                  <span data-l10n-id="editor_alt_text_mark_decorative_description">
+                  <span id="decorativeDescription" data-l10n-id="editor_alt_text_mark_decorative_description">
```

Comment:
Considering that this element is used as a label, and looking at the pre-existing `descriptionAreaLabel` above, how about the following name instead?
```suggestion
                  <span id="decorativeLabel" data-l10n-id="editor_alt_text_mark_decorative_description">
```